src/jvmMain/kotlin/io/github/moreirasantos/pgkn/PostgresDriver.kt

```diff
@@ -36,21 +64,25 @@ suspend fun PostgresDriver(
             .option(USER, user)
             .option(PASSWORD, password)
             .option(DATABASE, database)
+            .option(INITIAL_SIZE, poolSize)
             .option(MAX_SIZE, poolSize)
```

Comment:
Maybe we should modify our API to also support for initial-size, max-size, though that would make us change our pool implementation for kotlin-native.

Reply:
I thought about that, but I didn't want to change the API.
We can change it in another PR.